contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -42,29 +72,38 @@ void multisig::propose( ignore<name> proposer,
    std::vector<char> pkd_trans;
    pkd_trans.resize(size);
    memcpy((char*)pkd_trans.data(), trx_pos, size);
-   proptable.emplace( _proposer, [&]( auto& prop ) {
-      prop.proposal_name       = _proposal_name;
-      prop.packed_transaction  = pkd_trans;
-   });
-
-   approvals apptable( get_self(), _proposer.value );
-   apptable.emplace( _proposer, [&]( auto& a ) {
-      a.proposal_name       = _proposal_name;
-      a.requested_approvals.reserve( _requested.size() );
-      for ( auto& level : _requested ) {
-         a.requested_approvals.push_back( approval{ level, time_point{ microseconds{0} } } );
-      }
-   });
+
+   proptable.emplace( proposer, [&]( auto& prop ) {
+         prop.proposal_name      = proposal_name;
+         prop.packed_transaction = pkd_trans;
+         prop.earliest_exec_time = std::optional<time_point>{};
+      });
+
+   approvals apptable( get_self(), proposer.value );
+   apptable.emplace( proposer, [&]( auto& a ) {
+         a.proposal_name = proposal_name;
+         a.requested_approvals.reserve( requested.size() );
+         for ( auto& level : requested ) {
+            a.requested_approvals.push_back( approval{ level, time_point{ microseconds{0} } } );
+         }
+      });
 }
 
 void multisig::approve( name proposer, name proposal_name, permission_level level,
                         const eosio::binary_extension<eosio::checksum256>& proposal_hash )
 {
+   // if ( level.permission == "eosio.code"_n ) {
```

Comment:
Second commit says it was removing commented code, but this is still present.  Are we keeping it as independent reference?

Reply:
Removed commented out code.